orange/plugins/kvstore/api.lua

```diff
@@ -70,15 +70,6 @@ end
 
 local API = BaseAPI:new("kvstore-api", 2)
 
-API:get("/kvstore/configs", function(store)
```

Comment:
There are two `/kvstore/configs` in the same file.

The first one:
(from https://github.com/sumory/orange/blob/master/orange/plugins/kvstore/api.lua#L73)
```
API:get("/kvstore/configs", function(store)
    res:json({
        success = true, 
        data = {
            enable = orange_db.get("kvstore.enable")
        }
    })
end)
```

The second one:
(from https://github.com/sumory/orange/blob/master/orange/plugins/kvstore/api.lua#L227)
```
API:get("/kvstore/configs", function(store)
    return function(req, res, next)
        res:json({
            success = true,
            data = {
                enable = orange_db.get("kvstore.enable"),
                conf = orange_db.get_json("kvstore.conf")
            }
        })
    end
end)
```